docs/README.md

```diff
@@ -12,7 +12,7 @@ make html
 If you get warnings, like "WARNING: [autosummary] failed to import 'snowpark.dataframe': no module named snowpark.dataframe", try reinstalling the Snowpark API:
 
 ```bash
-python -m pip install ".[development, pandas]"
+python -m pip install ".[development, pandas, modin]"
```

Comment:
nit: this looks unrelated to vector type changes

Reply:
This is a bug we found, since we have included modin, the doc generator need modin dependencies to work

Reply:
Better use a different PR to address it.